Latex/inhalt/Kapitel1.tex

```diff
@@ -0,0 +1,12 @@
+%Alle Seiten beginnen mit der Oberüberschrift
+\section{Einleitung}
+Die ist eine PDF mit der Vorlage erstellt, um die Funktion und Formatierung dieser zu zeigen.
+
+Diese Vorlage\onlinezitat{SCZYRBA2020} ist ein Gemeinschaftsprojekt im Rahmen unseres Studiums.
+Der Docker-Container\onlinezitat{HILLE2021} gehört dazu.
+
+Weitere Hinweise befinden sich in der Readme.md oder im Wiki.
+Eine ausführliche Dokumentation zu diesem Dokument wird folgen.
+
+\section{Kapitel 1}
+Weitere Kapitel sind jetzt automatisch auf einer neuen Seite.
```

Comment:
Das hat in dieser Datei finde ich nichts zu suchen. Das ist eher Changelog.

Reply:
Das ist kein Changelog, das ist die Demonstration, dass es funzt :D